live-examples/js-examples/atomics-sub.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js" class="language-js">// create a SharedArrayBuffer with a size in bytes
+var buffer = new SharedArrayBuffer(16);
+var uint8 = new Uint8Array(buffer);
+uint8[0] = 7;
+
+Atomics.sub(uint8, 0, 2); // (returns 7)
+
+console.log(Atomics.load(uint8, 0)); // 7 - 2
+// expected output: 5
+</code>
```

Comment:
Nit: as for `add` above, consider logging the old value:

```
// create a SharedArrayBuffer with a size in bytes
var buffer = new SharedArrayBuffer(16);
var uint8 = new Uint8Array(buffer);
uint8[0] = 7;

// subtract 2 from the first element, and return the old value
console.log(Atomics.sub(uint8, 0, 2));
// expected output: 7

console.log(Atomics.load(uint8, 0)); // 7 - 2
// expected output: 5
```

Reply:
OK